src/ApparentHorizons/Tags.cpp

```diff
@@ -146,8 +147,7 @@ void RadiusCompute<Frame>::function(
     const ::Strahlkorper<Frame>& strahlkorper) {
   get(*radius).destructive_resize(
       strahlkorper.ylm_spherepack().physical_size());
-  get(*radius) =
-      strahlkorper.ylm_spherepack().spec_to_phys(strahlkorper.coefficients());
+  StrahlkorperFunctions::radius(radius, strahlkorper);
```

Comment:
The entire `RadiusCompute<Frame>::function` can now go away (and be replaced with `StrahlkorperFunctions::radius` in the hpp file).  But that can be another PR.  (It can be whichever PR adds more stuff to StrahlkorperFunctions, which I might do after this PR is merged because I need more functions there for for size control stuff).

Reply:
OK, let's leave that for another PR